test/fairplay.test.js

```diff
@@ -1,9 +1,9 @@
 import QUnit from 'qunit';
```

Comment:
Should we have tests for the original newer Fairplay system?

Reply:
Yeah com.apple.fps is well covered in https://github.com/videojs/videojs-contrib-eme/blob/main/test/eme.test.js